manager/routes/routes.go

```diff
@@ -113,7 +113,7 @@ func InitAdmin(app *gin.Engine, enableTurnpikeAuth bool) {
 	api.GET("/sessions", admin.GetActiveSessionsHandler)
 	api.GET("/sessions/:search", admin.GetActiveSessionsHandler)
 	api.DELETE("/sessions/:pid", admin.TerminateSessionHandler)
-	api.GET("/repack/:table_name", admin.RepackHandler)
+	api.GET("/repack/:table_name/*columns", admin.RepackHandler)
```

Comment:
shouldn't this be `/repack/:table_name/:columns` since `columns` is required according to spec? If I understand this correctly, `*columns` acts as optional parameter so the handler will also match `/repack/table_name/` and `/repack/table_name` which we actually don't need

Reply:
I believe `/repack/:table_name/*columns` is correct. OpenAPI does not allow optional path params ([OpenAPI-Specification](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/2.0.md#fixed-fields-7)), so the spec says `columns` are required even though they are not. Additionally, when tested locally, only `/repack/table_name/` and `/repack/table_name/columns` were valid and returned 200, `/repack/table_name` did not.

Reply:
An alternative would be to implement it as a query param, or to make the param required and change the implementation accordingly. So, can it stay like this or should I change it somehow?

Reply:
An alternative would be to implement it as a query param, or to make the param required and change the implementation accordingly. So, can it stay like this or should I change it somehow?